lib/fizzy/execute.cpp

```diff
@@ -343,6 +343,12 @@ inline bool trunc(OperandStack& stack) noexcept
     return false;
 }
 
+template <typename SrcT, typename DstT>
+inline void convert(OperandStack& stack) noexcept
+{
+    stack.top() = static_cast<DstT>(stack.top().as<SrcT>());
```

Comment:
It's getting difficult to see which of these small helpers are suitable for which types. Maybe we should add `static_assert`  here that `SrcT` is integer and `DstT` is floating point?

Reply:
That's why I used `FloatT` and `IntT` in tests. I will try doing something about it here.

Reply:
These static asserts for types are nice, we should have them in each helper where it makes sense.